timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -49,21 +51,40 @@ public long slowLockLogTriggerMillis() {
         return 10000;
     }
 
-    @JsonProperty("auth-clients")
+    /**
+     * The namespace and token credentials the server should use to authenticate basic service clients
+     */
+    @JsonProperty("service-auth-clients")
+    @Value.Default
+    public List<TimelockServiceAuthClient> serviceAuthClients() {
+        return ImmutableList.of();
+    }
+
+    /**
+     * The tokens the server should use to authenticate admin clients
+     */
+    @JsonProperty("admin-auth-clients")
     @Value.Default
-    public List<TimelockAuthClient> authClients() {
+    public List<String> adminAuthSecrets() {
         return ImmutableList.of();
     }
 
-    @JsonProperty("admin-token")
+    /**
+     * The token used to authenticate requests between internal nodes of the cluster if it's the case
+     */
+    @JsonProperty("internal-auth-secret")
     @Value.Default
-    public Optional<String> adminToken() {
+    public Optional<String> internalAuthSecret() {
         return Optional.empty();
     }
 
     @Value.Check
     public void check() {
         Preconditions.checkState(slowLockLogTriggerMillis() >= 0,
                 "Slow lock log trigger threshold must be nonnegative, but found %s", slowLockLogTriggerMillis());
+
+        Preconditions.checkState(
```

Comment:
We should also add this check to `TimelockServiceAuthClient.token()`.

Reply:
done